fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -133,6 +134,9 @@
     @VariableMgr.VarAttr(name = ENABLE_SPILLING)
     public boolean enableSpilling = false;
 
+    @VariableMgr.VarAttr(name = ENABLE_EXCHANGE_NODE_PARALLEL_MERGE)
+    public boolean enableExchangeNodeParallelMerge = true;
```

Comment:
default should be false?
If these are multi sort runs, but each run has a few rows, maybe this will increase the overhead.

Reply:
yes，I will change the default value.